express-routes/server.ts

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+
+const app = express();
+
+app.use((req, res, next) => {
+  console.log(
+    `The date is ${new Date()} the method is ${req.method}; the path is ${
+      req.path
+    }`
+  );
+  next();
+});
+
+app.get('/', (req, res, next) => {
+  res.send('second middleware reached');
+  next();
```

Comment:
No need to call `next()` if you send a response.

It doesn't hurt, though, and there are rare use cases where you would want to do it. But I would suggest you not do it in code you write at LFZ since it is not needed in anything we do.